css/team.css

```diff
@@ -1,64 +1,155 @@
-/* background */
-.team{
+.team {
     background-color: #fff;
-    height: 100%;
     width: 100%;
 }
 
-.teamDescription{
+.team__title {
     color: #324B4C;
     font-family: 'Roboto', sans-serif;
-    font-size: 1rem;
+    font-size: 2rem;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.team__person {
     text-align: center;
-    padding: 0 auto;
 }
 
-.teamName{
+.team__person--name {
     color: #324B4C;
     font-family: 'Roboto', sans-serif;
     font-size: 1.1rem;
     text-align: center;
     text-transform: capitalize;
 }
 
-.teamPhoto{
+.team__person--description {
+    color: #324B4C;
+    font-family: 'Roboto', sans-serif;
+    font-size: 1rem;
+    text-align: justify;
+    padding: 0 auto;
+}
+
+.team__clip-path-container {
+    display: inline-block;
+    position: relative;
+    height: 22vh;
+    width: 22vh;
+    margin: 10px ;
+    padding: 3px;
+    cursor: pointer;
+    transition: all .2s ease-in-out;
+    overflow: hidden;  
     border-radius: 50%;
-    width: 100%;
+    border-collapse: separate; 
+    background-color: #222;
 }
 
-.team__title{
-    color: #324B4C;
-    font-family: 'Roboto', sans-serif;
-    font-size: 2rem;
+.team__clip-path-container:focus,
+.team__clip-path-container:active,
+.team__clip-path-container:hover {
+    border-radius: 50%;
+    outline: none;
+} 
+
+.team__person--photo {
+    height: 22vh;
+    width: 22vh;
+    background-size: cover;
+    transition: all .2s ease-in-out;
+    transform: scale(1.4); 
```

Comment:
poprzednie skalowanie (1.1?) wyglądało korzystniej

Reply:
Jak wylacze skalowanie to nie bedzie efektu oddalania, wiec jak uwazacie?

Reply:
Ja bym dała 1.2, wygląda nieźle i nadal jest delikatny efekt oddalania